Formula/dpkg.rb

```diff
@@ -15,6 +15,7 @@ class Dpkg < Formula
     sha256 "9bf757d4e0e3902bbbc97a28a2532ac1a3c8220ad487c5a18a38925483e43062" => :yosemite
   end
 
+  depends_on :perl => "5.20"
```

Comment:
@DomT4 it's so unlike you to put a run-time dependency first.

Reply:
It'd be long enough since I've added one of these that honestly I couldn't remember quite where I usually stick 'em 😅.